README.md

```diff
@@ -63,10 +59,65 @@ On release, automated continuous integration tests run the pipeline on a full-si
                                  --outdir ./results
    ```
 
+## Pipeline summary
+
+### Default Steps
+
+By default the pipeline currently performs the following
```

Comment:
All steps are listed below additional steps. Shouldn't there be a few default steps?

Reply:
Apparently I removed this with the enthusiasm that I had.
I included this again!

Reply:
Perfect, looks great now!